src/apiCalls/getProjects.js

```diff
@@ -0,0 +1,32 @@
+import { refreshToken } from "./refreshToken";
+import { searchAction } from "../redux/actions/searchActions";
+
+export function getProjects(value, navigate) {
+  return async (dispatch) => {
+    const response = await fetch(
+      `http://localhost:3001/projects?value=${value}`,
```

Comment:
feel free to extract such values to env variables too https://create-react-app.dev/docs/adding-custom-environment-variables/, e.g. when you will build your project on heroku there will be different url